concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,146 @@
+# About 
+
+Package [regexp][package-regexp] offers support for regular expressions in Go.
+
+## Syntax 
+
+The [syntax][regexp-syntax] of the regular expressions accepted is the same general syntax used by Perl, Python, and other languages. 
+
+Both the search patterns and the input texts are interpreted as UTF-8.  
+
+When using backticks ``(`)`` to make strings, backslashes `(\)`  don't have any special
+meaning, and don't mark the beginning of special characters like tabs `\t` or
+newlines `\`n:
+
+```go
+"\t\n" // regular string literal with 2 characters: a tab and a newline
+`\t\n`// raw string literal with 4 characters: two backslashes, a 't', and an 'n'
+```
+
+Because of this, using backticks is desirable to make regular expressions,
+because it means we don't need to escape backslashes:
+
+```go
+"\\" // string with a single backslash
+`\\` // string with 2 backslashes
+```
+
+## Type `RegExp`
+
+The package defines a type `Regexp` for compiled regular expressions.
+Function `regexp.Compile` compiles a string into a regular expression.
+The function returns `nil` and an error if compilation failed:
```

Comment:
```suggestion
To use a regular expression, we first must compile the string pattern.
Compilation here means taking the string pattern of the regular expression and converting it into an internal representation that is easier to work with.
We only need to compile each pattern once, after that we can use the compiled version of the regular expression many times.
The type `regexp.Regexp` represents a compiled regular expressions.
We can compile a string pattern into a `regexp.Regexp` using the function  `regexp.Compile`.
This function returns `nil` and an error if compilation failed:
```

I think this clarification is important. I remember when learning regexes, one of the things that most confused me was the need for a compilation step, so I think we should try to address this. 
Another thing I see people new to regexes struggle with is that they may accpet need to compile their regex, but then they think they need to compile it every time they need to use it.